src/Post/postsController.js

```diff
@@ -1,10 +1,21 @@
+
 import postsService from "./postsService.js"
 
 class PostsController {
+    async getAll(req, res) {
+        try {
+            const posts = await postsService.getAll()
+            return res.json(posts)
+        }
+        catch (e) {
+            res.status(500).json(e)
+        }
+    }
     async create(req, res) {
+       // const {title, body} = req.body
```

Comment:
Delete comments and console logs please if you don't need them

Reply:
deleted